### PR TITLE
SOF-2271 Send commands back to Skaarhoj panel

### PR DESCRIPTION
**Pull request by LindvedKrvang**
Note: Dependent on https://github.com/tv2/sofie-tv2-server/pull/55

Able to send "commands" back to the Skaarhoj panel.
For now we can "turn buttons on/off" (should the show light or not), set the color of a button and set text in the display fields.

Added `color: Color` and `label: InputLabel` to `BaseInputConfiguration`, so that it's possible to set the color and text of inputs.

"SkaarCommands" has been introduced. This is a way to abstract away the actual string that is sent to Skaarhoj (it could look like `HWCc#12=132`). Instead we get to work with nice classes (the earlier string is the same as `new SkaarhojColorCommand('12', Color.RED)`

### CODE REVIEW

File: src/business-logic/panel-integrations/skaarhoj-command.ts [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  BLUE = '139',
+  CYAN = '141',
+  GREEN = '196'
+}
+
+export abstract class SkaarhojCommand {
+  protected constructor(protected id: string) {}
+
+  public abstract toString(): string
+}
```
Comment (KvelaGorrrrnio) [inline src/business-logic/panel-integrations/skaarhoj-command.ts:32]: Just a thought: Couldn't this be an interface like:
```typescript
export interface SkaarhojCommand {
  readonly id: string
  toString(): string
}
```
And is the `id` necessary for all commands?

**Review by KvelaGorrrrnio — APPROVED** [review]

**Review by RasmusAlbrektsen — APPROVED** [review]

File: src/business-logic/panel-integrations/skaarhoj-command.ts (hunk 1 above)
Comment (LindvedKrvang) [reply src/business-logic/panel-integrations/skaarhoj-command.ts:32]: The id is the identifer of the input the command is going to "modify". How would you go about identifying the input without some form of id?

For the interface thing: In the current implementation it currently doesn't matter that much whether it's an interface or abstract class.

File: src/business-logic/panel-integrations/skaarhoj-panel.ts [hunk 2]
```diff
@@ -244,6 +260,7 @@ export class SkaarhojPanel extends Panel {
         return this.updateTBarCommandWithValues(inputConfiguration.command, value)
       }
     }
+    return
```
Comment (MadsThy) [inline src/business-logic/panel-integrations/skaarhoj-panel.ts:263]: i don't think i see what this return does 😄

**Review by MadsThy — APPROVED** [review]

File: src/business-logic/panel-integrations/skaarhoj-panel.ts (hunk 2 above)
Comment (LindvedKrvang) [reply src/business-logic/panel-integrations/skaarhoj-panel.ts:263]: The return type of the method isn't `void`. When the return type isn't `void` the method has to have a return statement.

**Pull request merged**